src/Goerli-DssSpell.sol

```diff
@@ -69,9 +69,7 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
         // https://forum.makerdao.com/t/reth-collateral-onboarding-risk-evaluation/15286
 
         // Includes changes from the DssSpellCollateralAction
-        onboardCollaterals();
-        // updateCollaterals();
-        // offboardCollaterals();
+        collateralEngineeringAction();
```

Comment:
not sure how i feel about calling it "Engineering", maybe `collateralAction()` or `collateralSpellAction`?

Reply:
changed at 444a6575246675f3f3a587c87c373a334d3e6d14